### PR TITLE
chore: add forest-night theme

### PR DESCRIPTION
**Pull request by jef**
## Proposed Changes

- Adds [Forest Night](https://github.com/sainnhe/forest-night) theme

image

## Checklist

- [ ] Tested on all platforms changed
	- [x] Windows
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users

---

I figured you'd like to take a screenshot and add it to the list (or not). Up to you! Perhaps we also move this out of SCM in general and keep a wiki page up-to-date.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
 
 ## [Unreleased]
+### Added
+- Forest Night theme
```
Comment (casperstorm) [inline CHANGELOG.md:12]: Since we aren't currently shipping themes with the binary we shouldn't add an entry to the `CHANGELOG.md`.

**Review by casperstorm — CHANGES_REQUESTED** [review]
Hi jef,
Thanks for this PR. I would like you to take the screenshot. The one you have attached to this PR is just fine!

**Comment by jef** [comment]
> I would like you to take the screenshot. The one you have attached to this PR is just fine!

Great! I'll do you one better when I get on my mac. Thanks!

**Comment by casperstorm** [comment]
Lets go with Regular!

**Comment by casperstorm** [comment]
@jef You should add an entry to THEMES.md as well 🙂 otherwise it looks ready to go.

**Comment by casperstorm** [comment]
@jef take a look at https://github.com/casperstorm/ajour/pull/219.
I have been working on a refactor of the themes. I think it would make sense to work towards that with your theme instead. It will however require you add a few more colors but it will look better and we will ship Ajour with a set of themes and would love to have yours included.

Do you mind trying to redo towards the new system?

**Comment by jef** [comment]
Yeah! Let's do it. What branch are you working on so I can see your changes and adjust.

**Comment by casperstorm** [comment]
> Yeah! Let's do it. What branch are you working on so I can see your changes and adjust.

feat/refactor-color-themes

**Comment by casperstorm** [comment]
@jef branch is ready now - you can see how we add the new pre-baked themes here: https://github.com/casperstorm/ajour/blob/feat/refactor-color-themes/crates/core/src/theme/mod.rs#L322-L345

**Comment by jef** [comment]
Great! I'll make the changes and change the base ref to that branch.

**Comment by casperstorm** [comment]
I've marked it as draft, until it is ready @jef.

**Comment by jef** [comment]
> I've marked it as draft, until it is ready @jef.

Sorry and thank you. Works busy 🥲

**Comment by jef** [comment]
@casperstorm, not sure if you like it, but I also moved the themes so that the Dark and Light are grouped at the top, Alliance and Horde are group second to top, and the rest are below -- all in lexicographical order. Not sure if you like that or not. If not, I can revert those changes and we can just keep `Forest Night`.

**Comment by casperstorm** [comment]
@jef this is AWESOME.
I'll merge this. If you are looking for more work to do then I would love to get `monokai` theme as well eventually.
Or of you wanna dig deeper just let me know and we will find you a issue.

**Review by casperstorm — APPROVED** [review]

**Pull request merged**